classes/rawemailmessage.class.inc.php

```diff
@@ -582,7 +582,8 @@ protected function ExtractHeadersAndRawBody($aLines)
 			{
 				if (isset($aRawFields[$sCurrentHeader]))
 				{
-					$aRawFields[$sCurrentHeader] .= substr($sLine, 1);
+					// Fix for long subjects where spaces get lost on split header lines.
+					$aRawFields[$sCurrentHeader] .= (in_array($sCurrentHeader, array('references', 'subject')) ? $sLine : substr($sLine, 1));
```

Comment:
```suggestion
					// Fix for long values where spaces get lost on split header lines.
					$aRawFields[$sCurrentHeader] .= $sLine;
```

Reply:
@piRGoif would this need more investigation or can we just apply it to all headers?

Reply:
@piRGoif ?

Reply:
Hello,
Sorry I'm already in charge of lots of PR, I cannot handle this one right now :/

Reply:
@accognet there was still an open question here..

Reply:
The code _substr($sLine, 1)_ allow to remove \t characters in the beginning of some lines. I prefer to keep the PR with test on type of header, in order to better control the modification. I suppose that this _substr_ was not added by chance.

Reply:
So the best fix would actually be like this?
```suggestion
					// Fix for long subjects where spaces get lost on split header lines.
					$aRawFields[$sCurrentHeader] .= ltrim($sLine, "\t");
```